### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by thanhlqno1**
## Related Tickets
- [#42609](https://edu-redmine.sun-asterisk.vn/issues/42609)

## WHAT (optional)
 Add sign up and view profile features.

## Evidence (Screenshot or Video)
Sign up failed
image
Sign up successed
image

### CODE REVIEW

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the users controller here.
```
Comment (vanvtt-0952) [inline app/assets/stylesheets/users.scss:1]: Xóa file

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = t("flash.signup.successed")
+      redirect_to user_path(id: @user.id)
+    else
+      flash.now[:danger] = t("flash.signup.failed")
+      render "new"
```
Comment (vanvtt-0952) [inline app/controllers/users_controller.rb:21]:
```suggestion
      render :new
```

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (vanvtt-0952) [inline app/helpers/users_helper.rb:1]: Xóa file

File: app/models/user.rb [hunk 4]
```diff
@@ -7,14 +7,18 @@ class User < ApplicationRecord
...
   validates :name, presence: true, length: {maximum: Settings.length.digit_50}
   validates :email, presence: true,
             length: {maximum: Settings.length.digit_100},
             format: {with: Settings.email_regrex}, uniqueness: true
   validates :password, presence: true, allow_nil: true,
             length: {minimum: Settings.length.digit_6}
+
   has_secure_password
 
+  ATR_PERMIT = %i(name email password password_confirmation).freeze
```
Comment (vanvtt-0952) [inline app/models/user.rb:20]: Hằng số thì để trên line 2 hì

**Comment by thanhlqno1** [comment]
ready

**Pull request merged**